inffast_tpl.h

```diff
@@ -184,12 +184,13 @@ void Z_INTERNAL INFLATE_FAST(PREFIX3(stream) *strm, uint32_t start) {
             op = here->op;
             if (op & 16) {                      /* distance base */
                 dist = here->val;
-                op &= 15;                       /* number of extra bits */
+                op &= MAX_BITS;                 /* number of extra bits */
                 if (bits < op) {
                     hold |= load_64_bits(in, bits);
                     in += 6;
                     bits += 48;
                 }
+                op &= MAX_BITS;                 /* number of extra bits */
```

Comment:
There is no explanation why this line was added, and we generally discourage mixing cleanup and code changes.

Reply:
Good catch, thank you. Fixed.